app/src/main/java/com/yacgroup/yacguide/PartnersActivity.kt

```diff
@@ -139,8 +140,9 @@ class PartnersActivity : AppCompatActivity() {
 
     private fun _displayContent() {
         setTitle(R.string.partner)
+        supportActionBar?.setHomeAsUpIndicator(R.drawable.ic_baseline_save_24)
```

Comment:
There is also a predefined icon `android.R.drawable.ic_menu_save`.

Reply:
But it looks not so nice. I always prefer the material design.